All_home_works/hw2/task1.py

```diff
@@ -0,0 +1,96 @@
+"""
+Given a file containing text. Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+import string
+from collections import Counter, deque
+from typing import List
+
+TRANSLATE = str.maketrans({char: "" for char in string.punctuation})
+
+
+def _get_lines(file_path: str) -> str:
+    with open(file_path, encoding="raw_unicode_escape") as reading_file:
+        for line in reading_file.readlines():
+            line = line[:-1]
```

Comment:
We already have a discussion about such code in the different PR. Please check it.